cmd/node/main.go

```diff
@@ -159,6 +160,13 @@ VERSION:
 		Usage: "The port on which the rest API will start on",
 		Value: "8080",
 	}
+
+	// flag for monitoring using Prometheus
+	prometheus = cli.BoolFlag{
```

Comment:
If this is a bool flag, we should maybe call it something like `with-prometheus` or `use-prometheus`

Reply:
done